docs/introduction.rst

```diff
@@ -41,17 +41,20 @@ Are there other features, models or organisms that you'd like to see in ``stdpop
 This software is maintained by the PopSim Consortium,
 an open, global community of scientists and developers who are working together to improve
 standards for benchmarking and testing in population genetics.
-You can get into contact with the ``stdpopsim`` community by subscribing to our email list
-`serve <https://lists.uoregon.edu/mailman/listinfo/popgen_benchmark>`_
+You can get into contact with the ``stdpopsim`` community by subscribing to our `email list
+serve <https://lists.uoregon.edu/mailman/listinfo/popgen_benchmark>`_
 and by creating and commenting on
-Github `issue <http://github.com/popgensims/stdpopsim/issues>`_.
+Github `issues <http://github.com/popgensims/stdpopsim/issues>`_.
 There is a lot of chatter through
 Github, and we’ve been building code
 there cooperatively. If you are interested in joining us, as we very much hope you
 are, please read our
 `code of conduct <https://github.com/popsim-consortium/stdpopsim/blob/master/CODE_OF_CONDUCT.md>`_.
 A particular goal of this project is to build a community which is representative of population
-and evolutionary genetics writ large. 
+and evolutionary genetics as a field. We especially encourage individuals from underrepresented groups
+in to join. Being a member of the Consortium can provide a valuable avenue for career development, through
```

Comment:
"from underrepresented groups in population genetics to join"

Reply:
👍